tsdb/engine/tsm1/compact.go

```diff
@@ -681,6 +682,9 @@ type Compactor struct {
 	// RateLimit is the limit for disk writes for all concurrent compactions.
 	RateLimit limiter.Rate
 
+	// Returns the filename used for files generated by CompactFast().
+	CompactFastFormatFileName FormatFileNameFunc
```

Comment:
No need to separate out `Fast` type compactions.  All the names have to be the same.